src/libs/SidebarUtils.ts

```diff
@@ -120,23 +120,23 @@ function getOrderedReportIDs(
     policies: Record<string, Policy>,
     priorityMode: ValueOf<typeof CONST.PRIORITY_MODE>,
     allReportActions: OnyxCollection<ReportAction[]>,
+    transactionViolations: OnyxCollection<TransactionViolation[]>,
 ): string[] {
+    const reportIDKey = `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${currentReportId}`;
+    const reportActionCount = allReportActions?.[reportIDKey]?.length ?? 1;
```

Comment:
This changes logic. If length is 0, it returns 1 in old logic (0 || 1) but returns 0 in new logic (0 ?? 1)

Reply:
Fixed